packages/wallets/src/features/accounts/screens/PersonalDetails/PersonalDetails.tsx

```diff
@@ -1,89 +1,100 @@
-                    label='Citizenship*'
-                    list={residenceList.map(residence => ({
-                        text: residence.text as ReactNode,
-                        value: residence.value ?? '',
-                    }))}
-                    name='wallets-personal-details__dropdown-citizenship'
-                    onSelect={selectedItem => setFormValues('citizenship', selectedItem)}
-                    value={getSettings?.citizen ?? formValues?.citizenship}
-                    variant='comboBox'
-                />
-                <WalletDropdown
-                    disabled={getSettings?.place_of_birth !== ''}
-                    label='Place of birth*'
-                    list={residenceList.map(residence => ({
-                        text: residence.text as ReactNode,
-                        value: residence.value ?? '',
-                    }))}
-                    name='wallets-personal-details__dropdown-pob'
-                    onSelect={selectedItem => setFormValues('placeOfBirth', selectedItem)}
-                    value={getSettings?.place_of_birth ?? ''}
-                    variant='comboBox'
-                />
-                <WalletDropdown
-                    errorMessage={'Tax residence is required'}
-                    isRequired
-                    label='Tax residence*'
-                    list={residenceList.map(residence => ({
-                        text: residence.text as ReactNode,
-                        value: residence.value ?? '',
-                    }))}
-                    name='wallets-personal-details__dropdown-tax-residence'
-                    onChange={inputValue => {
-                        setFormValues('taxResidence', inputValue);
-                    }}
-                    onSelect={selectedItem => setFormValues('taxResidence', selectedItem)}
-                    value={getSettings?.tax_residence ?? formValues?.taxResidence}
-                    variant='comboBox'
-                />
-                <FlowTextField
-                    defaultValue={getSettings?.tax_identification_number ?? formValues?.taxIdentificationNumber}
-                    errorMessage={'Please fill in tax residence'}
-                    isInvalid={!formValues.taxResidence || !formValues.taxIdentificationNumber}
-                    label='Tax identification number*'
-                    name='taxIdentificationNumber'
-                />
-                <WalletDropdown
-                    label='Account opening reason*'
-                    list={accountOpeningReasonList}
-                    name='wallets-personal-details__dropdown-opening-reason'
-                    onSelect={selectedItem => setFormValues('accountOpeningReason', selectedItem)}
-                    value={getSettings?.account_opening_reason ?? formValues?.accountOpeningReason}
-                    variant='comboBox'
-                />
-            </div>
+            {isLoading && <Loader />}
```

Comment:
I just added a loading screen when the default values are not yet loaded, this scenario only happens when we show POA screen initially
<img width="1840" alt="Screenshot 2023-11-23 at 4 04 02 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/9dbc4693-021e-488b-9898-76423c679ad8">

Reply:
<img width="553" alt="Screenshot 2023-11-24 at 1 05 25 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/c85fd2c6-2733-4188-a193-8e0ad010394a">